core/src/main/resources/jenkins/install/SetupWizard/setupWizardFirstUser.jelly

```diff
@@ -1,55 +1,52 @@
-<!--
-  This is used to create the first user.
--->
 <?jelly escape-by-default='true'?>
-<l:html norefresh="true" title="${it.displayName}" xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
 
-  <l:main-panel>
+  <l:layout norefresh="true" type="full-screen" title="${it.displayName}">
```

Comment:
@kzantow I think we should not use `iframes`. I would like to work on a proposal based on a wizard (flow) with _real_ steps / states, where we can have a URL per state.